scripts/typeGeneration/convert.js

```diff
@@ -0,0 +1,135 @@
+import { basename, dirname, join, resolve } from 'https://deno.land/std/path/posix.ts';
+import { parse, stringify } from 'https://deno.land/std/yaml/mod.ts';
+import { camelCase } from 'https://deno.land/x/case/mod.ts';
+import { generate } from 'npm:openapi-typescript-codegen';
+import converter from 'npm:swagger2openapi';
+
+const apiTypes = ['management'];
```

Comment:
wondering aloud: should we think of something similar with internal and device endpoints specs? the reason I ask is: this approach will eventually clean-up management APIs but will it not be in slight discord with the rest?